### ISSUE TITLE
Navigation by keyboard (by tab and arrows)

### ISSUE THREAD

**Issue by SzajekSzymon** [Knotx/knotx-fragments-chrome-extension#69]
**Is your feature request related to a problem? Please describe.**

currently keyboard navigation doesn't work correctly. User has access to non visible elements.

**Describe the solution you'd like**

Create accessibility improvement to allow navigate by keyboard through the application.
